app/_src/kubernetes-ingress-controller/support-policy.md

```diff
@@ -124,25 +132,26 @@ LTS versions of {{site.kic_product_name}} are supported for 3 years after releas
 
 ## {{site.kic_product_name}} versions
 
-| Version  | Release Date  | End of Full Support | End of Sunset Support |
-|:--------:|:-------------:|:-------------------:|:---------------------:|
-|  2.11.x  |  2023-08-09   |     2024-08-09      |      2025-08-09       |
-|  2.10.x  |  2023-06-02   |     2024-06-02      |      2025-06-02       |
-|  2.9.x   |  2023-03-09   |     2024-03-09      |      2025-03-09       |
-|  2.8.x   |  2022-12-19   |     2023-12-19      |      2024-12-19       |
-|  2.7.x   |  2022-09-27   |     2023-09-27      |      2024-09-27       |
-|  2.6.x   |  2022-09-15   |     2023-09-15      |      2024-09-15       |
-|  2.5.x   |  2022-07-11   |     2023-07-11      |      2024-07-11       |
-|  2.4.x   |  2022-06-15   |     2023-06-15      |      2024-06-15       |
-|  2.3.x   |  2022-04-05   |     2023-04-05      |      2024-04-05       |
-|  2.2.x   |  2022-02-04   |     2023-02-04      |      2024-02-04       |
-|  2.1.x   |  2022-01-05   |     2023-01-05      |      2024-01-05       |
-|  2.0.x   |  2021-10-07   |     2022-10-07      |      2023-10-07       |
-|  1.3.x   |  2021-05-27   |     2022-05-27      |      2024-05-27       |
-|  1.2.x   |  2021-03-24   |     2022-03-24      |      2024-03-24       |
-|  1.1.x   |  2020-12-09   |     2021-12-09      |      2023-12-09       |
-|  1.0.x   |  2020-10-05   |     2021-10-05      |      2023-10-05       |
-|  0.x.x   |  2018-06-02   |     2019-06-02      |      2020-06-02       |
+| Version | Release Date | End of Full Support | End of Sunset Support |
+|:-------:|:------------:|:-------------------:|:---------------------:|
+| 2.12.x  |  2023-09-25  |     2024-09-25      |      2025-09-25       |
```

Comment:
@mheap  Given that 2.12.x is meant to be an LTS release, should we make the EOL to be `2026-09-25`?